check50.py

```diff
@@ -42,8 +43,14 @@ def main():
 
     # check for newer version on PyPi
     pypi = pypijson.get("check50")
-    if pypi and not args.force and StrictVersion(pypi["info"]["version"]) > VERSION:
-        raise RuntimeError("You are running an old version of check50. Run pip install check50 --upgrade, and then run check50 again!")
+    if pypi and not args.no_autoupdate and StrictVersion(pypi["info"]["version"]) > VERSION:
+
+        # updade check50
+        pip = "pip3" if sys.version_info >= (3, 0) else "pip"
+        subprocess.run([pip, "install", "--upgrade", "check50"])
+        check50 = os.path.realpath(__file__)
+        os.execv(check50, sys.argv + ["--no-autoupdate"])
+        raise RuntimeError("Impossible.")
```

Comment:
Is it conventional to have an arbitrary line of code like that, though? As opposed to nothing (or just a comment)?